src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domain
+
+object LottoFactory {
+    fun create(
+        autoTicketCount: Int,
+        manualLottoNumbers: List<String>,
+        manualLottoDelimiter: String = ",",
+    ): List<Lotto> {
+
+        val lottos = mutableListOf<Lotto>()
+        for (index in 0 until autoTicketCount) {
+            lottos += Lotto.auto()
+        }
```

Comment:
```suggestion
        val lottos = (1..autoTicketCount).map { Lotto.auto() } 
```
스타일 차이긴 하겠지만 최대한 함수형으로 풀어내려고 노력하는 편입니다